src/main/webapp/dev/js/common/instructorEnroll.js

```diff
@@ -38,7 +38,7 @@ function getUpdatedHeaderString(handsontableColHeader) {
 function getUserDataRows(spreadsheetData) {
     // needs to check for '' as an initial empty row with null values will be converted to e.g. "||||" after .map
     return spreadsheetData.filter(row => (!row.every(cell => cell === null || cell === '')))
-            .map(row => row.join('|'))
+            .map(row => row.join('|').replace(/\n|\r/g, ''))
```

Comment:
For every method in the chain, shouldn't we start a new line?
```
.map()
.replace()
.join()
```

In addition, do we have test to cover the cases?

Reply:
> For every method in the chain, shouldn't we start a new line?

@xpdavid The current way works but yes I agree the suggested way is easier to read. Will make the change :)

> In addition, do we have test to cover the cases?

@damithc suggested to release a hotfix first so right now I haven't added tests for this case. However I did experimented a bit and adding tests might be a little tricky. 

As of now, I can only replicate this issue by copying data from `.xlsx` files and pasting it in the `New Students` spreadsheet. Copying and pasting from Google Sheets would not cause this issue.